include/deal.II/fe/fe_interface_values.h

```diff
@@ -190,11 +229,318 @@ namespace FEInterfaceViews
+     * the type used to store the values of the unknowns $U_j$ of your finite
+     * element vector $U$ (represented by the @p fe_function argument).
+     *
+     * @dealiiRequiresUpdateFlags{update_hessians}
+     */
+    template <class InputVector>
+    void
+    get_jump_in_function_hessians(
+      const InputVector &fe_function,
+      std::vector<solution_hessian_type<typename InputVector::value_type>>
+        &hessians) const;
+
+    /**
+     * This function relates to get_jump_in_function_hessians() in the same way
+     * as get_function_values_from_local_dof_values() relates to
+     * get_function_values(). See the documentation of
+     * get_function_values_from_local_dof_values() for more
+     * information.
+     */
+    template <class InputVector>
+    void
+    get_jump_in_function_hessians_from_local_dof_values(
+      const InputVector &local_dof_values,
+      std::vector<solution_hessian_type<typename InputVector::value_type>>
+        &hessians) const;
+
+    /**
+     * Return the jump in the third derivatives of the selected scalar component
+     * of the finite element function characterized by <tt>fe_function</tt> at
+     * the quadrature points of the cell, face or subface selected the last time
+     * the <tt>reinit</tt> function of the FEInterfaceValues object was called.
+     *
+     * The data type stored by the output vector must be what you get when you
+     * multiply the third derivatives of shape functions (i.e., @p
+     * third_derivative_type) times the type used to store the values of the
+     * unknowns $U_j$ of your finite element vector $U$ (represented by the @p
+     * fe_function argument).
+     *
+     * @dealiiRequiresUpdateFlags{update_third_derivatives}
+     */
+    template <class InputVector>
+    void
+    get_jump_in_function_third_derivatives(
+      const InputVector &fe_function,
+      std::vector<
+        solution_third_derivative_type<typename InputVector::value_type>>
+        &third_derivatives) const;
+
+    /**
+     * This function relates to get_jump_in_function_third_derivatives() in the
+     * same way as get_function_values_from_local_dof_values() relates
+     * to get_function_values(). See the documentation of
+     * get_function_values_from_local_dof_values() for more
+     * information.
+     */
+    template <class InputVector>
+    void
+    get_jump_in_function_third_derivatives_from_local_dof_values(
+      const InputVector &local_dof_values,
+      std::vector<
+        solution_third_derivative_type<typename InputVector::value_type>>
+        &third_derivatives) const;
+
   private:
     /**
      * The extractor for this view.
      */
     const FEValuesExtractors::Scalar extractor;
+
+    /**
+     * Get the local degree-of-freedom values associated with the internally
+     * initialized cell interface.
+     */
+    template <class InputVector, class OutputVector>
+    void
+    get_local_dof_values(const InputVector &dof_values,
```

Comment:
Could this go in the `Base` class, so that it doesn't have to be implemented twice?

Reply:
make senses!